### PR TITLE
Fix bug of not being able to build with -DGO_USE_JACK=OFF

### PR DESCRIPTION
**Pull request by larspalo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Fix bug preventing building with -DGO_USE_JACK=OFF
```
Comment (oleg68) [inline CHANGELOG.md:1]:
```suggestion
- Fixed bug preventing building with -DGO_USE_JACK=OFF
```

**Review by oleg68 — APPROVED** [review]

**Comment by oleg68** [comment]
@rousseldenis please review this PR

**Comment by oleg68** [comment]
@larspalo it is a time to lern resolving conflicts. Please click on the `Resolve conflicts` button and make the final version of CHANGELOG.md

**Comment by oleg68** [comment]
> > @larspalo it is a time to lern resolving conflicts. Please click on the `Resolve conflicts` button and make the final version of CHANGELOG.md
> 
> Personnally, I don't like this way of resolving conflict.

But it is the fastest and the easiest way. 

It is very usefule especially for CHANGELOG.md, because conflicts on it appear very often. Resolving them should be as soon as possible, otherwise another conflicts with the same file may occur.

And, as @larspalo mentioned, this way is available from everywhere, even from a smartphone.

**Comment by oleg68** [comment]
> Or is there any way I can do the commands you suggest directly on my fork on GitHub (not on my local copy which isn't available at the moment)?

Only a part of them is available.

**Comment by oleg68** [comment]
> In the project I've implied, a git bot is used in order to merge and do some extra stuffs.
> 
> We could imagine in the future a same behaviour. For instance, add a line in Changelog automatically with the PR title or ...

Please start a separate discussion on it. Not here.

**Review by rousseldenis — APPROVED** [review]

**Comment by larspalo** [comment]
> The merge commit will appear in destination branch after the PR merge.

And doesn't it explain what has happened?

**Comment by rousseldenis** [comment]
> @rousseldenis I can agree that it's problematic that you have to approve once again as reviewers with the resolve on GitHub. If doing it your way with rebase eliminates that, I'll try to stick to that procedure whenever I have access to my own computer.

I think this is an option. I will see

**Pull request merged**